examples/dtls-sock/dtls-client.c

```diff
@@ -122,22 +122,22 @@ static int client_send(char *addr_str, char *data, size_t datalen)
         return -1;
     }
 
-    res = sock_dtls_session_create(&dtls_sock, &remote, &session);
+    res = sock_dtls_session_create(&dtls_sock, &remote, &session, SOCK_NO_TIMEOUT);
     if (res < 0) {
         printf("Error creating session: %d\n", (int)res);
         sock_dtls_close(&dtls_sock);
         sock_udp_close(&udp_sock);
         return -1;
     }
 
-    if (sock_dtls_send(&dtls_sock, &session, data, datalen) < 0) {
+    if (sock_dtls_send(&dtls_sock, &session, data, datalen, 0) < 0) {
         puts("Error sending data");
     }
     else {
         printf("Sent DTLS message\n");
 
         uint8_t rcv[512];
-        if (sock_dtls_recv(&dtls_sock, &session, rcv, sizeof(rcv), SOCK_NO_TIMEOUT) < 0) {
+        if (sock_dtls_recv(&dtls_sock, &session, rcv, sizeof(rcv), 0) < 0) {
```

Comment:
Is it a good idea to use no timeout for both functions here? For send I understand, as the session is already created, but not for recv.

Reply:
You're right. I'll change this to 3 seconds.